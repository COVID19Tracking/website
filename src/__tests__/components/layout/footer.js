import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '../../../components/layout/footer'

describe('Components : Layout : Footer', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
