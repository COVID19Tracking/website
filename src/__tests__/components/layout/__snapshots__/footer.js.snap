// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Layout : Footer renders correctly 1`] = `
<footer
  className="wrapper"
>
  <div
    className="footer"
  >
    <div
      className="main"
    >
      <a
        className="ctpLogo"
        href="/"
      >
        <img
          alt="The COVID Tracking Project"
          className="projectLogo"
          src="test-file-stub"
        />
      </a>
      <div
        className="links"
      >
        <ul
          className="footerList"
        >
          <li>
            <a
              href="/contact"
            >
              Contact
            </a>
          </li>
          <li>
            <a
              href="/terms-and-conditions"
            >
              Terms and Conditions
            </a>
          </li>
          <li>
            <a
              href="/about-data/license"
            >
              License
            </a>
          </li>
          <li>
            <a
              href="https://www.theatlantic.com/privacy-policy/"
            >
              Privacy Policy
            </a>
          </li>
          <li>
            <a
              href="/about/accessibility"
            >
              Accessibility
            </a>
          </li>
          <li>
            <a
              href="/rss.xml"
            >
              RSS
            </a>
          </li>
        </ul>
      </div>
      <ul
        className="social"
      >
        <li>
          <a
            href="https://github.com/COVID19Tracking"
          >
            <img
              alt="GitHub"
              src="test-file-stub"
            />
          </a>
        </li>
        <li>
          <a
            href="https://twitter.com/COVID19Tracking"
          >
            <img
              alt="Twitter"
              src="test-file-stub"
            />
          </a>
        </li>
      </ul>
    </div>
  </div>
  <hr
    className="divider"
  />
  <div
    className="copyright"
  >
    <span>
      CovidTracking.com Copyright Â© 
      2020
       by The Atlantic Monthly Group. 
      <a
        href="/license"
      >
        (CC BY 4.0)
      </a>
    </span>
    <a
      className="backToTop"
      href="#reach-skip-nav"
    >
      <span>
        Back to top
      </span>
      <img
        alt=""
        aria-hidden={true}
        src="test-file-stub"
      />
    </a>
  </div>
</footer>
`;
